index.js

```diff
@@ -125,19 +139,23 @@ class Markdown extends Component {
         );
     }
 
-    renderText(node, key, extras) {
+    renderText(node, key, extras, textType) {
 
         const { styles } = this.state;
 
         let style = (extras && extras.style) ? [styles.text].concat(extras.style) : styles.text;
 
         if (node.props) {
+            // Hmm
```

Comment:
If they have their own text renderer, and we've received the array of Elements from their renderers, should we just wrap them in a fragment and return?

That way their consuming function only ever takes 1 node at a time and doesnt have to receive an array of elements? Or should `this.props.renderText` always receive an array of elements and we let the user solve this problem?